src/components/TheEntityPanel.vue

```diff
@@ -66,42 +75,61 @@
 import { Component, Prop, Vue } from 'vue-property-decorator'
 import { RoomGetters, RoomState } from '@/store/modules/room'
 import { Getter } from 'vuex-class'
+import { WowsPanel, WotPanel } from './entity-panel'
+
+export interface MenuItem {
+  title: string;
+  icon: string;
+  color?: string;
+  noOfEntities?: number;
+}
 
 @Component({
-  name: 'TheEntityPanel'
+  name: 'TheEntityPanel',
+  components: {
+    WowsPanel,
+    WotPanel
+  }
 })
 export default class MapButtons extends Vue {
   @Prop() private id!: string;
-  @Getter(`room/${RoomGetters.ROOM_STATE}`) roomState!: RoomState
+  @Getter(`room/${RoomGetters.ROOM_STATE}`) roomState!: RoomState;
 
   show = false
```

Comment:
put an explicit scope on properties (public, protected, private) and set them to readonly if they don't get updated

Reply:
sure!